### PR TITLE
More Chaplain Clothes

### PR DESCRIPTION
**Pull request by Aurorablade**
Vine Jesu Domini
Bonae est requiem

[thwack]

Ports witch hunter outfit from https://github.com/tgstation/tgstation/pull/22109

Adds monk robe, whitch is a coder sprite recolor from me..no its not good.

:cl: Fethas
rscadd: Chaplains have some new clothes in thier locker.
/ :cl:

### CODE REVIEW

**Comment by KingPhilipIII** [comment]
Because chappie didn't have enough stuff already. Smh.

**Comment by KasparoVy** [comment]
Different clothes for different... Religions?

**Comment by Aurorablade** [comment]
@KingPhilipIII i was do lazy to put it into other areas..someone else could do it if they wanted..

**Comment by Fox-McCloud** [comment]
Thou shalt not suffer a witch to live.

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -54,6 +54,21 @@
...
 
+//Chaplain
+/obj/item/clothing/head/monk_hood
+	name = "monk hood"
+	desc = "Wooden board not included."
+	icon_state = "monk_hood"
+	flags = BLOCKHAIR
+	flags_cover = HEADCOVERSEYES
+
+/obj/item/clothing/head/witchunter_hat
```
Comment (SamHPurp) [inline code/modules/clothing/head/jobs.dm]: `witchhunter` not `witchunter`, surely?

File: code/modules/clothing/head/jobs.dm [hunk 2]
```diff
@@ -54,6 +54,21 @@
...
+	name = "witchunter hat"
```
Comment (SamHPurp) [inline code/modules/clothing/head/jobs.dm]: Same here

**Comment by taukausanake** [comment]
> Chaplain is a job that's 100% about gimmicks

Saying that implies they have no functionality during cult or vampire rounds, the latter of which is argued already I believe. Doesn't really sound fair to me

**Pull request merged**